stack.yaml

```diff
@@ -79,3 +79,184 @@ nix:
   # Disabling the pure nix-shell allows environment variables to be
   # passed down to tests. We need this for integration tests.
   pure: false
+
+# !!! This can be dropped when cabal-3.4.0.0 is released !!!
```

Comment:
Seems quite wrong to have the cabal config spreading like this :/ 
Couldn't this be figured out automatically by stack2cabal?

Reply:
How is this wrong? Otherwise you have to do it manually. This is a "bug" in cabal-install and a workaround has no place in stack2cabal. That's exactly what this verbatim feature is for.